geoportailv3/static/less/geoportailv3.api.less

```diff
@@ -1 +1,77 @@
 @import (inline) '../../../node_modules/openlayers/css/ol.css';
+@import "../../../node_modules/bootstrap/less/mixins/vendor-prefixes.less";
+
+
+@popup-bg: #fff;
+@popup-border-color: rgba(0,0,0,0.2);
+@popup-arrow-outer-color: @popup-border-color;
+@popup-arrow-color: @popup-bg;
+@popup-arrow-outer-width: 11px;
+@popup-arrow-width: 11px;
+.lux-popup {
+  background-color: @popup-bg;
+  background-clip: padding-box;
+  border: 1px solid @popup-border-color;
+  border-radius: 5px;
+  .box-shadow(0 5px 10px rgba(0,0,0,.2));
+  z-index: 2;
+
+  & > .lux-popup-arrow {
```

Comment:
Nesting simplified.
I also removed the 'lux-popup-top' rule since there's only one possible option for positioning.